File: testdata/jennies/rawtypes/package-with-dashes/JavaRawTypes/withdashes/RefreshRate.java
package withdashes;


// Refresh rate or disabled.
public class RefreshRate {
    protected String string;
    protected Boolean bool;
    protected RefreshRate() {}
    public static RefreshRate createString(String string) {
        RefreshRate refreshRate = new RefreshRate();
        refreshRate.string = string;
        return refreshRate;
    }
    public static RefreshRate createBool(Boolean bool) {
        RefreshRate refreshRate = new RefreshRate();
        refreshRate.bool = bool;
        return refreshRate;
    }
}

File: testdata/jennies/rawtypes/package-with-dashes/CSharpRawTypes/src/Grafana/Foundation/WithDashes/RefreshRate.cs
namespace Grafana.Foundation.WithDashes;


// Refresh rate or disabled.
public class RefreshRate
{
    public string String;
    public bool Bool;

    public RefreshRate()
    {
    }

    public RefreshRate(string stringArg, bool boolArg)
    {
        this.String = stringArg;
        this.Bool = boolArg;
    }
}
